test/e2e/apimachinery/resource_quota.go

```diff
@@ -1194,7 +1194,7 @@ var _ = SIGDescribe("ResourceQuota", func() {
 		err = wait.PollUntilContextTimeout(ctx, 5*time.Second, 6*time.Minute, true, func(ctx context.Context) (bool, error) {
 			resourceQuotaResult, err := rqClient.Get(ctx, rqName, metav1.GetOptions{})
 			if err != nil {
-				return false, nil
+				return false, err
```

Comment:
retrying the GET and returning the error SGTM, instead of failing the test.
one problem is that if the server is not responding this would block for 6 minutes.

this error check could be broken down and managed better with failing on specific server responses, but that can be difficult to prepare.

Reply:
I thought the same, but Jordan is right, and I checked and we fail on unexpected apiserver errors.

The client-go already retries "retriable" errors, this looks correct to me

Reply:
if this is supposed to "error" here, then it should be `return true, err`, not an assert inside the poll IMO.

Reply:
`return true, err` and `return false, err` are equivalent if `err != nil`